jwst/tweakreg/tests/test_tweakreg.py

```diff
@@ -21,6 +29,41 @@ def dummy_source_catalog():
     return catalog
 
 
+@pytest.mark.parametrize("inplace", [True, False])
+def test_rename_catalog_columns(dummy_source_catalog, inplace):
+    renamed_catalog = tweakreg_step._rename_catalog_columns(dummy_source_catalog)
+
+    # if testing inplace, check the input catalog
+    if inplace:
+        catalog = dummy_source_catalog
+    else:
+        catalog = renamed_catalog
+
+    assert 'xcentroid' not in catalog.colnames
+    assert 'ycentroid' not in catalog.colnames
+    assert 'y' in catalog.colnames
+    assert 'y' in catalog.colnames
```

Comment:
Did you mean `x` in one of the above 2 lines?

Reply:
Yes! Good catch.